spec/lib/spree_sitemap/spree_defaults_spec.rb

```diff
@@ -27,6 +27,18 @@
     end
   end
 
+  context ".gem_available?" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.